README.md

```diff
@@ -29,9 +29,9 @@ You can install this awesome theme through the [Package Control](https://package
 # Activate the theme
 
 You can active this theme from:
-- Command palette `Tools > Command Palette` (or <kbd>cmd/ctrl</kbd> + <kbd>↑</kbd> + <kbd>p</kbd>) by typing `Material Theme: Activate theme`.
-- Context menu (Right click on the editor) and choose `Material Theme > Activate Material Theme`
-- `Preferences > Packages Settings` and choose `Material Theme > Activate Material Theme`
+- Command palette `Tools > Command Palette` (⌘/Ctrl + ⇧ + P) by typing `Material Theme: Activate theme`.
```

Comment:
I removed the `<kbd>` tags since they compressed and mangled the keyboard symbols. I can add them back if you'd like.